### PR TITLE
Updated Rollingstone fallback fonts

### PR DESCRIPTION
**Pull request by prajapatisagar**
### Doneness Checklist:

Pre-release # (or n/a):

- [ ] Updated root CHANGELOG.md with summary of changes under `Unpublished` section
- [ ] Ran the visual regression tests manually (see [the README](https://github.com/penske-media-corp/pmc-larva#running-visual-regression-tests) for details)
- [ ] `npm run prod` in this repo outputs expected changes (excepting the issue with re-ordered partials in larva-css algorithms partials - see [LRVA-1885](https://jira.pmcdev.io/browse/LRVA-1885))
- [ ] If changes to build scripts or the Node.js server, tested changes in pmc-spark [via a pre-release](https://confluence.pmcdev.io/x/XhOeAw)
- - If changes to build tools: npm scripts `prod`, `lint`, and `dev` scripts run as expected
- - If changes to Larva server: Static site generates as expected in a theme  (avail. on a URL {brand}.stg.larva.pmcdev.io)

### CODE REVIEW

File: packages/larva-tokens/src/brands/rollingstone-2022.json [hunk 1]
```diff
@@ -194,42 +195,64 @@
...
+        "name": "FONT_FAMILY_BODY"
+      },
+	  "FONT_FAMILY_BODY_FANCY": {
+        "category": "font-family",
+        "value": "'Georgia', Helvetica, sans-serif",
+        "type": "font",
+        "originalValue": "'Georgia', Helvetica, sans-serif",
+        "name": "FONT_FAMILY_BODY"
+      },
+      "FONT_FAMILY_PRIMARY": {
```
Comment (laras126) [inline packages/larva-tokens/src/brands/rollingstone-2022.json:225]: @prajapatisagar The FONT_FAMILY_PRIMARY token should be a websafe fallback font - in this case, probably Georgia. 

For FONT_FAMILY_BODY, it is not using a webfont that needs to be loaded externally, so both the FONT_FAMILY_BODY and FONT_FAMILY_BODY_FANCY tokens can be the same.

The way you have FONT_FAMILY_SECONDARY is correct - you could specify a specific websafe typeface (see [a list here](https://www.w3schools.com/cssref/css_websafe_fonts.asp)), but it's also acceptable to only have serif or sans-serif.

Comment (laras126) [reply packages/larva-tokens/src/brands/rollingstone-2022.json:225]: Also - this file will need to have the values added for font sizes, which you can add with the form, but I think it makes sense to wait until we have the Desktop XL breakpoint to do that.

See [this file for an example](https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-tokens/src/brands/robbreport.json#L172) of what it looks like to have all of those added.

**Review by laras126 — CHANGES_REQUESTED** [review]
A change needed to add the fallback font for Primary. You can think of it as "What font should display while the webfont is loading?"

[This post describes](https://css-tricks.com/loading-web-fonts-with-the-web-font-loader/) the approach we use for webfont loading with fallback fonts and the Flash Of Unstyled Text (FOUT) - it's something that is becoming an outdated practice, but we haven't updated our standards yet to account for the improved approach.

File: packages/larva-tokens/src/brands/rollingstone-2022.json (hunk 1 above)
Comment (prajapatisagar) [reply packages/larva-tokens/src/brands/rollingstone-2022.json:225]: @laras126 
The FONT_FAMILY_PRIMARY token should be a websafe fallback font - in this case, probably Georgia.
**Updated FONT_FAMILY_PRIMARY.**

For FONT_FAMILY_BODY, it is not using a webfont that needs to be loaded externally, so both the FONT_FAMILY_BODY and FONT_FAMILY_BODY_FANCY tokens can be the same.
**Changed value to Georgia**

The way you have FONT_FAMILY_SECONDARY is correct - you could specify a specific websafe typeface (see [a list here](https://www.w3schools.com/cssref/css_websafe_fonts.asp)), but it's also acceptable to only have serif or sans-serif.
**As per my knowledge here not needed changes.**

Also - this file will need to have the values added for font sizes, which you can add with the form, but I think it makes sense to wait until we have the Desktop XL breakpoint to do that.
**Yes right we will update it once breakpoint added.**

**Review by laras126 — APPROVED** [review]
Wonderful, thank you Sagar!

**Pull request merged**